src/main/java/bio/overture/ego/repository/queryspecification/TokenStoreSpecification.java

```diff
@@ -16,6 +16,34 @@
 
 package bio.overture.ego.repository.queryspecification;
 
+import static bio.overture.ego.model.enums.JavaFields.*;
+
 import bio.overture.ego.model.entity.ApiKey;
+import bio.overture.ego.model.entity.User;
+import bio.overture.ego.utils.QueryUtils;
+import java.util.UUID;
+import javax.persistence.criteria.Join;
+import lombok.NonNull;
+import lombok.val;
+import org.springframework.data.jpa.domain.Specification;
+
+public class TokenStoreSpecification extends SpecificationBase<ApiKey> {
+
+  public static Specification<ApiKey> containsText(@NonNull String text) {
```

Comment:
Maybe I'm misunderstanding, but wouldn't this be clearer if this method was called `containsName` or `containsTextInName` or something to that effect, as there is a description field for api keys that is also text/string based.